Cargo.toml

```diff
@@ -11,9 +11,6 @@ edition = "2018"
 readme = "changelog.md"
 include = ["src/**/*", "LICENSE.md", "README.md", "CHANGELOG.md"]
 
-[lib]
-test = false
```

Comment:
This has tripped me up many times: "WHY do my tests not run, what's going on" :). I hope you could find it quickly.